src/onepay-merchant-checkout.js

```diff
@@ -0,0 +1,147 @@
+import {styles} from 'merchant.css';
+const zoid = require('zoid');
+const Smartphone = require('./smartphone');
+
+// Scripts
+const BASE_URL = 'https://rawgit.com/TransbankDevelopers/transbank-sdk-js-onepay/feat/modal-iframe/html';
```

Comment:
This shouldn't be hardcoded. 

Here the `Onepay.version` variable will become useful. You can do:
```javascript
const BASE_URL = 'https://rawgit.com/TransbankDevelopers/transbank-sdk-js-onepay/v' + Onepay.version + '/html';
```

Reply:
Done